menpo/io/test/io_import_test.py

```diff
@@ -302,6 +302,24 @@ def test_importing_PIL_1_no_normalize(is_file, mock_image):
 
 @patch('PIL.Image.open')
 @patch('menpo.io.input.base.Path.is_file')
+def test_importing_PIL_1_proper_conversion(is_file, mock_image):
+    from menpo.image import BooleanImage
+ 
+    arr = np.zeros((10, 10), dtype=np.uint8)
+    arr[4, 4] = 255
```

Comment:
That's what the line 318 is doing with the np.all. If you print the values of arr (arr.astype(np.bool)) this value is True.